### ISSUE TITLE
Make email addresses interactive

### ISSUE THREAD

**Issue by tinashe404** [atsign-company/at_wavi_app#161]
**Is your feature request related to a problem? Please describe.**
If you are looking at someone's page in the app and they have an email address that they have entered, you have to copy and paste it elsewhere to use it.

**Describe the solution you'd like**
It would be great if pressing on an email address would prompt you to select which app you want to use to send them an email. After selecting the app, the email would pre-populated in the email.

**Comment by jalanprakhar** [comment]
I am very new to flutter and i am thinking of using the url_launcher package to solve this issue. Can you please tell me if i am on the right path and if yes can you please assign this to me ?

**Comment by yahu1031** [comment]
@jalanprakhar You can implement as optimized as possible. Please go ahead with your fix. We will look into your PR. Please ping us in discord if you are stuck somewhere. 😄 Happy coding.

**Comment by Ninad07** [comment]
Hi, I am Ninad, a contributor to OpenForce 2022. I would like to work on this issue. I would be making a PR as soon as I am done with resolving the issue. Thank you

**Comment by jalanprakhar** [comment]
> @jalanprakhar You can implement as optimized as possible. Please go ahead with your fix. We will look into your PR. Please ping us in discord if you are stuck somewhere. 😄 Happy coding. do you want me to assign it to you?

Yes please. Also I do have some doubts and have mentioned those in the @company's channel

**Comment by shubhamshah14102** [comment]
Hey @yahu1031 . I am Shubham Shah, a contributor to OpenForce 2022.
I would like to work on this issue so please can you assign it to me
